AmplifyPlugins/Core/AWSPluginsCore/API/AppSyncErrorType.swift

```diff
@@ -22,6 +23,8 @@ public enum AppSyncErrorType: Equatable {
 
     case unauthorized
 
+    case operationDisabled
```

Comment:
Can you add some documentation around this to indicate that customers should not use this unless they have consulted with AWS?   Here's the similar blurb I added on Android: https://github.com/aws-amplify/amplify-android/blob/f51cc5b5eea477a2ad0b34c2012de065c6051884/aws-datastore/src/main/java/com/amplifyframework/datastore/appsync/AppSyncExtensions.java#L147-L151

Reply:
thanks, updated